lib/server.js

```diff
@@ -12,4 +12,12 @@ const server = new Hapi.Server({
 
 server.connection({ port: Config.PORT });
 
+server.register([
+  require('happi-bookshelf-serializer')
```

Comment:
Extra `p` in `hapi`. It's not causing any problems yet because we're not using the plugin yet but it would cause bugs in your next PR.